Readme.md

```diff
@@ -18,151 +18,163 @@ The easiest way to use AFLGo is as patch testing tool in OSS-Fuzz. Here is our i
-export COPY_CFLAGS=$CFLAGS
-export COPY_CXXFLAGS=$CXXFLAGS
-export ADDITIONAL="-targets=$TMP_DIR/BBtargets.txt -outdir=$TMP_DIR -flto -fuse-ld=gold -Wl,-plugin-opt=save-temps"
-export CFLAGS="$CFLAGS $ADDITIONAL"
-export CXXFLAGS="$CXXFLAGS $ADDITIONAL"
-
-# Build libxml2 (in order to generate CG and CFGs).
-# Meanwhile go have a coffee ☕️
-export LDFLAGS=-lpthread
-pushd $SUBJECT
-  ./autogen.sh
-  ./configure --disable-shared
-  make clean
-  make xmllint
-popd
-# * If the linker (CCLD) complains that you should run ranlib, make
-#   sure that libLTO.so and LLVMgold.so (from building LLVM with Gold)
-#   can be found in /usr/lib/bfd-plugins
-# * If the compiler crashes, there is some problem with LLVM not 
-#   supporting our instrumentation (afl-llvm-pass.so.cc:540-577).
-#   LLVM has changed the instrumentation-API very often :(
-#   -> Check LLVM-version, fix problem, and prepare pull request.
-# * You can speed up the compilation with a parallel build. However,
-#   this may impact which BBs are identified as targets. 
-#   See https://github.com/aflgo/aflgo/issues/41.
-
-
-# Test whether CG/CFG extraction was successful
-$SUBJECT/xmllint --valid --recover $SUBJECT/test/dtd3
-ls $TMP_DIR/dot-files
-echo "Function targets"
-cat $TMP_DIR/Ftargets.txt
-
-# Clean up
-cat $TMP_DIR/BBnames.txt | grep -v "^$"| rev | cut -d: -f2- | rev | sort | uniq > $TMP_DIR/BBnames2.txt && mv $TMP_DIR/BBnames2.txt $TMP_DIR/BBnames.txt
-cat $TMP_DIR/BBcalls.txt | grep -Ev "^[^,]*$|^([^,]*,){2,}[^,]*$"| sort | uniq > $TMP_DIR/BBcalls2.txt && mv $TMP_DIR/BBcalls2.txt $TMP_DIR/BBcalls.txt
-
-# Generate distance ☕️
-# $AFLGO/scripts/genDistance.sh is the original, but significantly slower, version
-$AFLGO/scripts/gen_distance_fast.py $SUBJECT $TMP_DIR xmllint
-
-# Check distance file
-echo "Distance values:"
-head -n5 $TMP_DIR/distance.cfg.txt
-echo "..."
-tail -n5 $TMP_DIR/distance.cfg.txt
-```
-8) Note: If `distance.cfg.txt` is empty, there was some problem computing the CG-level and BB-level target distance. See `$TMP_DIR/step*`.
-9) Instrument subject (i.e., libxml2)
-```bash
-export CFLAGS="$COPY_CFLAGS -distance=$TMP_DIR/distance.cfg.txt"
-export CXXFLAGS="$COPY_CXXFLAGS -distance=$TMP_DIR/distance.cfg.txt"
-
-# Clean and build subject with distance instrumentation ☕️
-pushd $SUBJECT
-  make clean
-  ./configure --disable-shared
-  make xmllint
-popd
-```
+   ```bash
+   sudo apt-get update
+   sudo apt-get install python3
+   sudo apt-get install python3-dev
+   sudo apt-get install python3-pip
+   sudo apt-get install libboost-all-dev  # boost is not required if you use gen_distance_orig.sh in step 7
+   sudo pip3 install --upgrade pip
+   sudo pip3 install networkx
+   sudo pip3 install pydot
+   sudo pip3 install pydotplus
+   ```
 
-If your compilation crashes in this step, have a look at Issue [#4](https://github.com/aflgo/aflgo/issues/4#issuecomment-333947041).
+3) Compile AFLGo fuzzer, LLVM-instrumentation pass and the distance calculator
```

Comment:
Still need to set the environment variable. Copying from above:

```bash
# Checkout source code
git clone https://github.com/aflgo/aflgo.git
export AFLGO=$PWD/aflgo
```

Reply:
Solved in 7a204ca.